build.ps1

```diff
@@ -30,47 +30,12 @@ function bazel() {
 # which is a workaround for this problem.
 bazel shutdown
 
-# FIXME: Until all bazel issues on Windows are resolved we will be testing only specific bazel targets
-bazel build `-`-experimental_execution_log_file ${ARTIFACT_DIRS}/build_execution_windows.log `
-    //:git-revision `
-    @com_github_grpc_grpc//:grpc `
-    @haskell_c2hs//... `
-    //3rdparty/... `
-    //compiler/... `
-    //daml-assistant/... `
-    //daml-foundations/... `
-    //daml-lf/... `
-    //extractor/... `
-    //hazel/... `
-    //language-support/... `
-    //ledger/... `
-    //ledger-api/... `
-    //libs-haskell/... `
-    //navigator/... `
-    //nix/... `
-    //notices-gen/... `
-    //release/... `
-    //rules_daml/... `
-    //scala-protoc-plugins/... `
-    //templates/...
+bazel build `-`-experimental_execution_log_file ${ARTIFACT_DIRS}/build_execution_windows.log //...
 
 bazel shutdown
 
-bazel run `
-    //daml-foundations/daml-tools/da-hs-damlc-app `-`- `-h
+bazel run //daml-foundations/daml-tools/da-hs-damlc-app `-`- `-h
```

Comment:
I think we can just kill this line. We have tests that exercise this code path now, e.g, the integration tests. This was only added when we first got damlc to compile.